src/esn.inbox.libs/app/components/message-body/html/message-body-html.spec.js

```diff
@@ -9,6 +9,15 @@ describe('The inboxMessageBodyHtml component', function() {
   var $compile, $rootScope, $timeout, newComposerService;
   var element;
 
+  const tokenAPIMock = {
+    getNewToken: function() {
+      return $q.when({ data: { token: '123' } });
```

Comment:
what is 123 ?

Reply:
It doesn't matter the value here, just a token value

Reply:
this is a unit @tanandy